src/main/java/com/ezasm/gui/RegisterTable.java

```diff
@@ -42,6 +44,26 @@ public RegisterTable(Registers registers) {
         add(scrollPane);
     }
 
+    /**
+     *      * The function to highlight changed registers      
+     */
+    public void ChangeCellColor(int row_index) {
+        TableCellRenderer render = new DefaultTableCellRenderer() {
+            @Override
+            public Component getTableCellRendererComponent(JTable table, Object value, boolean isSelected,
+                    boolean hasFocus, int row, int col) {
+                final Component c = super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, col);
+                if (row == row_index || (row == 1 && row_index != -1)) {
+                    c.setForeground(Color.red);
```

Comment:
These should use theme colors -- not arbitrary ones.

Reply:
Then is it ok if I use the red in theme?